.github/workflows/ci.yml

```diff
@@ -22,6 +22,7 @@ jobs:
       - run: cargo clippy
       - run: ./panic_safety.sh
       - run: cargo test --verbose -- --ignored
+      - run: cargo test --no-default-features
```

Comment:
should we have `--verbose`, since we do for the other test commands?

Reply:
Sure.